client/my-sites/hosting/staging-site-card/sync-options-panel.tsx

```diff
@@ -23,7 +23,7 @@ const DangerousItemsTitle = styled.p( {
 
 const ToggleControlWithHelpMargin = styled( ToggleControl )( {
 	'.components-base-control__help': {
-		marginLeft: '4em',
+		marginLeft: '44px',
```

Comment:
@kozer I needed to use px to align the hint with the radio button label. Do you see a better solution for this component?